src/file-upload-with-preview.js

```diff
@@ -101,6 +100,26 @@ export default class FileUploadWithPreview {
 
                 this.deleteFileAtIndex(selectedFileIndex)
             }
+
+            // on image click
+            if (event.target.matches('.custom-file-container__image-multi-preview')) {
+                // Get its token
+                let fileToken = event.target.querySelector('.custom-file-container__image-multi-preview__single-image-clear__icon').getAttribute('data-upload-token')
+
+                // Get the index of the file
+                let selectedFileIndex = this.cachedFileArray.findIndex(x => x.token === fileToken)
+
+                // Send out our event
+                let imageClickedEvent = new CustomEvent('fileUploadWithPreview:imageClicked', {
```

Comment:
We'll want to update the `README` to indicate a new `event`. I got you though - I'll do this before I push the update.